Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM debian:12.0-slim AS build
+
+# copy script files
+WORKDIR /build
+COPY script /build/script
+
+# install development dependencies
+RUN export DEBIAN_FRONTEND="noninteractive" \
+    && apt-get update \
+    && apt-get upgrade --quiet --yes \
+    && apt-get install --quiet --yes --no-install-recommends \
+        gettext-base       `# provides envsubst` \
+        git                `# to extract commit info` \
+    && script/install_debian_dependencies.sh \
```

Comment:
calling the generic `script/install_dependencies.sh` would mean fewer changes if this was re-used for a different platform